packages/agent/app/agent.ts

```diff
@@ -2,8 +2,17 @@ import { parse } from 'bowser';
 import { QueryParams } from './query-params';
 import { Agent } from '../shared/agent';
 import { run } from './runner';
+import { Controller, Operation, OperationFn, Sequence } from 'effection';
```

Comment:
Everything but `Operation` here will not make it to the next version of Effection. All effection operations should be typed as `Operation<T>` where `T` is the return value of the operation.